compiler/rustc_codegen_llvm/src/debuginfo/metadata/enums/cpp_like.rs

```diff
@@ -490,6 +494,8 @@ fn build_variant_struct_wrapper_type_di_node<'ll, 'tcx>(
                 variant_index,
             ),
             &variant_struct_wrapper_type_name(variant_index),
+            unknown_file_metadata(cx),
+            UNKNOWN_LINE_NUMBER,
```

Comment:
It would be cool if we could also get the line numbers for the enum variants but that's a little more complex. If you want to take a stab at that, you need to get the `AdtDef` for the enum (like we do for structs) and then call the `variant` method, providing the `variant_index`. From there, you the `def_id` should work with `tcx.def_span`.